test/api/publish.test.js

```diff
@@ -0,0 +1,154 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+import supertest from 'supertest';
+
+import crypto from 'crypto';
+
+import { Storage as MockStorage } from '../../__mocks__/@google-cloud/storage';
+
+import { config } from '../../src/config';
+import { createApp } from '../../src/app';
+
+beforeEach(() => MockStorage.cleanUp());
+afterEach(() => MockStorage.cleanUp());
+
+describe('publishing endpoints', () => {
+  function setup() {
```

Comment:
Maybe this setup function should be named something like `getPreconfiguredRequest`. What do you think?

Reply:
I'm fine with either.